app/bundles/CoreBundle/Config/config.php

```diff
@@ -915,95 +916,96 @@
     ],
 
     'ip_lookup_services' => [
-        'freegeoip' => [
+        'freegeoip'         => [
```

Comment:
Can you please revert all the unnecessary space changes? It's the most of the changes of this PR. I'd suggest a simple rule to not touching code which you don't have to. It will make code reviews much faster.